src/operator/groupBy.ts

```diff
@@ -240,6 +240,8 @@ class GroupDurationSubscriber<K, T> extends Subscriber<T> {
       group.error(err);
     }
     this.parent.removeGroup(this.key);
+    this.unsubscribe();
+    this.parent.remove(this);
```

Comment:
@kwonoj if anything, the `GroupDurationSubscriber` should be a pass-through to the `group` Subject. If we did that, the base Subscriber `_error` and `_complete` implementations would call `unsubscribe`, and the Subscription's unsubscribe would automatically call `parent.remove()`